crates/revm/revm-inspectors/src/tracing/mod.rs

```diff
@@ -371,7 +373,7 @@ impl TracingInspector {
         step.gas_cost = step.gas_remaining - self.gas_inspector.gas_remaining();
 
         // set the status
-        step.status = status;
+        step.status = InstructionResult::Continue;
```

Comment:
Also not immediately obvious that this is correct. Previously we'd passed the state along here.